ml/src/main/scala/ai/h2o/sparkling/ml/params/H2OAlgoParamsHelper.scala

```diff
@@ -18,132 +18,6 @@ package ai.h2o.sparkling.ml.params
 
 import java.util
```

Comment:
nit: maybe rename this class to something like `EnumParamValidator` ?

Reply:
Np, will rename